### ISSUE TITLE
Automation Script - Compare With Maximo - Error

### ISSUE THREAD

**Issue by jaredschrag** [sharptree/vscode-autoscript-deploy#14]
I am using VSCode v1.88.1 with the extension @Sharptree.maximo-script-deploy v1.13.5 installed, and am connecting to an instance of Maximo v7.6.1.3 (IFIX018).

When it comes to Automation Scripts, the features "Deploy to Maximo" and "Extract Automation Script" seem to work as advertised. Nice work! However, if I try to perform "Compare with Maximo", it always fails with the following error:
![Sharptree_Error_CompareWithMaximo](https://github.com/sharptree/vscode-autoscript-deploy/assets/143452984/198479a3-3067-4306-98d4-be8d6e2ba958)

I get this error from an editor for a script that I've just extracted, or have only modified slightly. All of my scripts are python. It indicates to "see log for details"; however, below is the only thing logged in the Maximo server logs. Is there another place I should be looking for additional logging about this error?
```
18 Apr 2024 14:02:22:321 [ERROR] [MXServerA2] [CID-MXOSLC-11430] {}
18 Apr 2024 14:02:22:321 [ERROR] [MXServerA2] [CID-MXOSLC-11430] [object Object]
```

**Comment by jvenhuizen** [comment]
I have unfortunately not been able to reproduce this error although there are small differences as I have Maximo 7.6.1.3 (I don't don't have your exact iFix) and VS Code 1.89.1, but I doubt those are going to make a difference.

What is interesting is that to do the compare we are calling the extract function exactly as when you extract a script normally, we are then just using the inbuilt VS Code compare function, which does nothing to parse the script.

Can you check your SystemErr.log to see if there is an error logged there?  Any additional information would be appreciated.

**Comment by jvenhuizen** [comment]
With the recent updates can you reverify this issue?  As mentioned before there is nothing in the processing that should be parsing the script so I am unsure what could cause the issue.

**Comment by jvenhuizen** [comment]
Hi Jared,

Would you mind reverifying this issue given that there have been numerous updates.  If I don't hear back in a week I will close the issue, but feel free to reopen it if you are still experiencing a problem.

Jason

**Comment by jaredschrag** [comment]
Hello Jason,

After ensuring that the VSCode extension updated, as well as letting it deploy the updated Maximo configurations, I tried the VSCode extension "Compare with Maximo" feature. I still see the same results. I've included a screenshot of the error as well as a Maximo log file. Hope you find this to be helpful!
![2024-07-01_13-05-40](https://github.com/sharptree/vscode-autoscript-deploy/assets/143452984/b4ae7a9c-2ae4-4b97-9fe7-ff07bb7b943b)
[MXServer.log](https://github.com/user-attachments/files/16057614/MXServer.log)
